### PR TITLE
Lps 82330

### PR DESCRIPTION
**Pull request by shuyangzhou**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@mtambara can you add this to relevant?

@jpince this is the test we talked about @jhendley fyi

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/5dd836fb85...964887a906

**Comment by brianchandotcom** [comment]
@mtambara please backport. Thx.

File: test.properties [hunk 1]
```diff
@@ -1437,6 +1440,7 @@
         functional-smoke-tomcat90-mysql57-jdk8,\
         functional-tomcat90-mysql57-jdk8,\
         integration-mysql57-jdk8,\
+        lpkg-base-jdk8,\
```
Comment (shuyangzhou) [inline test.properties]: @brianchandotcom this line is for ci relevant.

**Comment by mtambara** [comment]
@brianchandotcom Do we want to have this same requirement for 7.0.x?

**Pull request closed without merging**